config/routes.rb

```diff
@@ -587,7 +588,8 @@ def scoped_navigation_routes(context, navigation)
 
       scope ':us_state', as: 'ny', constraints: { us_state: :ny } do
         scoped_navigation_routes(:questions, Navigation::StateFileNyQuestionNavigation)
-        resources :ny_w2, only: [:index, :edit, :update, :create], module: 'state_file/questions', path: 'questions/ny_w2'
+        # TODO: This can be deleted once no intake has w2 as the current step
```

Comment:
does "this" refer to the path being `ny_w2`?

Reply:
Fixed.